unity-renderer/Assets/DCLServices/Lambdas/PaginatedResponse.cs

```diff
@@ -6,9 +6,9 @@ namespace DCLServices.Lambdas
     [Serializable]
     public abstract class PaginatedResponse
     {
-        [SerializeField] internal int pageNum;
-        [SerializeField] internal int pageSize;
-        [SerializeField] internal int totalAmount;
+        [SerializeField] protected internal int pageNum;
+        [SerializeField] protected internal int pageSize;
+        [SerializeField] protected internal int totalAmount;
```

Comment:
Why did you need to make these variables protected?

Reply:
so i can initialize them in the `WearableWithDefinitionResponse` constructor.. i didnt find any way to set them in the tests